docs/acr42/01_introduction_and_goals.md

```diff
@@ -0,0 +1,68 @@
+This document follows the [arc42 template](https://arc42.org/).
+# Introduction and Goals
+
+This document describes LAPIS, which is a platform to give easy access to genomic sequence data alongside metadata of
+the sequenced probes. It is used to filter sequence data and return the result to the user through web access, so the
+user can develop his/her own evaluation of the data. The current development is based on two existing
```

Comment:
```suggestion
user can develop their own evaluation of the data. The current development is based on two existing
```
If we want to be gender neutral here, I think "their" is the preferred way in English.